configurator/src/lnd.conf.template

```diff
@@ -51,22 +51,25 @@ autopilot.conftarget={autopilot_confirmation_target}
 
 [tor]
 tor.active=true
-tor.socks={tor_proxy}
+tor.socks=embassy:9050
+tor.skip-proxy-for-clearnet-targets={tor_enable_clearnet}
+tor.streamisolation={tor_stream_isolation}
 
 [watchtower]
-watchtower.active={watchtower_enabled}
+watchtower.active=false
```

Comment:
does this mean we're removing watchtowers?

Reply:
For now, yes. They never worked. The work required to make them function properly is significant and ought to be done separately from the update release. It will also require quite a serious refactor to the current system to support with high reliability.